### PR TITLE
Active-window-tracker raises error when try to send servideMessage to removed iframe (fix #206)

### LINKED ISSUE #206: Active-window-tracker raises error when try to send servideMessage to removed iframe
Error is reproduced in testcafe-phoenix's Qunit tests only in MSEdge https://github.com/superroma/testcafe-phoenix/issues/110

Error occurs when we try to verify the existence of this.activeWindow.top in _notifyPrevActiveWindow method.

We just need to add this check in try/catch block (and we added addition check):
```js
_notifyPrevActiveWindow () {
        try {
            if (this.activeWindow.top && this.activeWindow !== this.activeWindow.top) {
                this.sandbox.message.sendServiceMsg({
                    cmd: WINDOW_DEACTIVATED_EVENT
                }, this.activeWindow);
            }
        }
        catch (err) {
            //NOTE: the error appears in IE when the corresponding iframe is removed
            return void 0;
        }
    }
```

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @miherlosev

### CODE REVIEW

**Comment by churkin** [comment]
\cc @AlexanderMoskovkin

**Comment by AlexanderMoskovkin** [comment]
lgtm overall, but I suppose we should check the strings @VasilyStrelyaev

**Comment by churkin** [comment]
lgtm

**Comment by miherlosev** [comment]
lgtm

File: test/client/fixtures/sandbox/event/active-window-tracker-test.js [hunk 1]
```diff
@@ -92,3 +92,41 @@ asyncTest('check switching active window (between two iframes)', function () {
             });
     });
 });
+
+asyncTest('check that error do not rises when try to send serviceMessage to removed iframe', function () {
```
Comment (inikulin) [inline test/client/fixtures/sandbox/event/active-window-tracker-test.js]: Should be in regression section with the appropriate test name format

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
FPR?

**Comment by LavrovArtem** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

File: test/client/fixtures/sandbox/event/active-window-tracker-test.js [hunk 2]
```diff
@@ -92,3 +92,43 @@ asyncTest('check switching active window (between two iframes)', function () {
             });
     });
 });
+
+module('regression');
+
+asyncTest('check that error do not rises when try to send serviceMessage to removed iframe (GH-206)', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/sandbox/event/active-window-tracker-test.js]: check that an error does not rise when trying to send serviceMessage to the removed iframe (GH-206)

**Comment by VasilyStrelyaev** [comment]
lgtm

**Pull request merged**